docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,160 @@
+
+4. Content designer to:
+    - write announcements to post on Slack after we release:
+      - GOV.UK Frontend (for example, [draft comms for the cookie banner component](https://docs.google.com/document/d/1jVyMB7i94NOeflWaf3kE4Q4APMXGfluK3rOh74IHO08/edit))
+      - GOV.UK Prototype kit
+    - check who the release’s contributors are and if we have consent to include their name
+
+5. Technical writer to finalise draft of release notes and get 2i on them.
+
+6. Content designer or designer to update community backlog with rationale for any decisions we made.
+
+7. Person leading the release to coordinate the sign-off when they are ready to do the release. Once the team agrees, this confirms a code and content freeze. Use the [#design-system-team-channel](https://gds.slack.com/app_redirect?channel=design-system-team-channel) to confirm sign-off from:
+    - content designer, technical writer and designers for guidance, examples and community backlog decision rationale
+    - technical writer and developers for Nunjucks macros
+    - developers for changes to GOV.UK Frontend
+    - technical writer for release notes
+    - content designer/Community manager/Technical writer for announcements and engagement activities
+
+Note: Before you go on leave, tell the delivery manager who will be looking after your work. This will help us to complete sign-off.
+
+### Publish a new version of GOV.UK Frontend from the support branch
+
+#### Change the code
+
+1. To check out the support branch for the current major release, run `git checkout support/<CURRENT MAJOR VERSION NUMBER>.x`. If the branch does not exist, follow these steps to create it:
+
+    - make sure you have all tags locally by running `git fetch --all --tags --prune`
+    - run `git checkout tags/v<LAST RELEASED VERSION NUMBER> -b support/<CURRENT MAJOR VERSION NUMBER>.x` - for example, `git checkout tags/v3.9.1 -b support/3.x`
+
+2. Run `nvm use` to make sure you’re using the right version of Node.js and npm.
+
+3. Push the support branch to GitHub. The branch will automatically have branch protection rules applied.
+
+4. To fix the issue, create a new branch (for example, `git checkout -b fix-the-thing`) from the `support/<CURRENT MAJOR VERSION NUMBER>.x` branch.
+
+5. Run `npm install` to make sure you have the latest dependencies installed.
+
+6. Make your code changes, and test them following our [standard testing requirements](/docs/contributing/testing.md).
+
+7. Update the changelog with details of the fix.
+
+8. Commit your changes, then push your new branch (see step 4) to GitHub and raise a pull request, with `support/<CURRENT MAJOR VERSION NUMBER>.x` as the base branch to merge into.
+
+9. Once the PR is approved, merge it into `support/<CURRENT MAJOR VERSION NUMBER>.x`. It’s usually a developer who reviews the PR, but sometimes PRs need an extra review from another role.
```

Comment:
Recommend saying who would approve this PR to remove passive e.g. "Once XYZ has approved the PR, merge the PR into..."